arangod/Replication2/ReplicatedLog/InMemoryLog.cpp

```diff
@@ -213,6 +213,14 @@ auto replicated_log::InMemoryLog::getIteratorFrom(LogIndex fromIdx) const
   return std::make_unique<ReplicatedLogIterator>(std::move(log));
 }
 
+auto replicated_log::InMemoryLog::getMemtryIteratorFrom(LogIndex fromIdx) const
```

Comment:
Should that read `getMemoryIteratorFrom`?

Reply:
😆 a `Memtry` is a **Mem**ory En**try** ... I thought this would be funny.